src/http/frame/headersframe.rs

```diff
@@ -0,0 +1,621 @@
+//! The module contains the implementation of HTTP/2 frames.
```

Comment:
It seems this is where the original module-level doc string ended up -- this one should be in `frame/mod.rs` :)